qualtran/bloqs/data_loading/select_swap_qrom.py

```diff
@@ -252,10 +315,53 @@ def wire_symbol(self, reg: Optional[Register], idx: Tuple[int, ...] = tuple()) -
             trg_indx = int(name.replace('target', '').replace('_', ''))
             # match the sel index
             subscript = chr(ord('a') + trg_indx)
-            return TextBox(f'data_{subscript}')
+            return TextBox(f'QROAM_{subscript}')
         elif name == 'control':
             return Circle()
         raise ValueError(f'Unknown register name {name}')
 
     def _value_equality_values_(self):
-        return self.block_size, self._target_bitsizes, self.data
+        return self.log_block_sizes, *super()._value_equality_values_()
+
+
+@bloq_example
+def _qroam_multi_data() -> SelectSwapQROM:
+    data1 = np.arange(5)
+    data2 = np.arange(5) + 1
+    qroam_multi_data = SelectSwapQROM.build_from_data([data1, data2])
+    return qroam_multi_data
+
+
+@bloq_example
+def _qroam_multi_dim() -> SelectSwapQROM:
+    data1 = np.arange(25).reshape((5, 5))
+    data2 = (np.arange(25) + 1).reshape((5, 5))
+    qroam_multi_dim = SelectSwapQROM.build_from_data([data1, data2])
+    return qroam_multi_dim
+
+
+@bloq_example
+def _qroam_symb_dirty() -> SelectSwapQROM:
```

Comment:
I think for the purposes of exposition it would be helpful to add a 1D example with the symbols matching those from the screenshotted costs.